src-tauri/src/util.rs

```diff
@@ -80,3 +81,15 @@ pub fn dedupe_paths(paths: Vec<PathData>) -> Vec<PathData> {
         .filter(|path| seen.insert(path.raw.clone()))
         .collect()
 }
+
+pub fn create_openhome_directory(app_handle: &tauri::AppHandle) -> Result<(), String> {
+    
+    let appdata_dir = get_appdata_dir(app_handle)?;
+    let mut full_path = PathBuf::new();
+
+    full_path.push(appdata_dir);
+    full_path.push("storage".to_owned());
+
+    create_dir_all(full_path).unwrap();
+    Ok(())
```

Comment:
Instead of calling `unwrap` here, can you return it as a result? Otherwise failing to create the directory would cause a panic instead of returning a descriptive error

Reply:
Sure thing